Dockerfile.rocm

```diff
@@ -52,10 +76,11 @@ COPY --from=build_hipblaslt ${COMMON_WORKDIR}/hipBLASLt/build/release/*.deb /
 FROM scratch AS export_hipblaslt_0
 FROM export_hipblaslt_${BUILD_HIPBLASLT} AS export_hipblaslt
 
+
 # -----------------------
 # RCCL build stages
 FROM base AS build_rccl
-ARG RCCL_BRANCH="73221b4"
+ARG RCCL_BRANCH="833435b"
```

Comment:
This looks like it's past the point they introduced a bug with queues.
Let's make this an official release tag, probably dependent on rocm version

Reply:
I don't see any issues with `rocm/pytorch:rocm6.2_ubuntu20.04_py3.9_pytorch_staging` as the base image.